src/domain/request/usecase/ExecuteReadyRequestsUseCase.spec.ts

```diff
@@ -1,17 +1,17 @@
 import { expect } from 'chai';
 import { describe, it } from 'mocha';
 
-import SilentLogger from '@core/application/logger/SilentLoggerAdapter';
-import InMemoryResponseRepositoryAdapter from '@core/infrastructure/persistence/inmemory/InMemoryResponseRepositoryAdapter';
+import SilentLogger from '../../../application/logger/SilentLoggerAdapter';
```

Comment:
Why removed @core from path?

Reply:
Because TSLint was complaining about this. Turns out it's just my local TSLint config so putting it back.